Library/Formula/pilot-link.rb

```diff
@@ -0,0 +1,22 @@
+class PilotLink < Formula
+  homepage "http://www.pilot-link.org/"
+  url "http://downloads.pilot-link.org/pilot-link-0.12.5.tar.gz"
+  version "0.12.5"
+  sha256 "159b84cadf5cdb39e238feaa652e98fc289a5624356a8ddd7f84d6a55eb5c736"
+
+  depends_on "popt" => :build
+  depends_on "libpng12" => :build
```

Comment:
On 27. mars 2015 16:12, Dominyk Tiller wrote:
>
> In Library/Formula/pilot-link.rb 
> <https://github.com/Homebrew/homebrew/pull/38120#discussion_r27301898>:
>
> > @@ -0,0 +1,22 @@
> > +class PilotLink < Formula
> > +  homepage "http://www.pilot-link.org/"
> > +  url "http://downloads.pilot-link.org/pilot-link-0.12.5.tar.gz"
> > +  version "0.12.5"
> > +  sha256 "159b84cadf5cdb39e238feaa652e98fc289a5624356a8ddd7f84d6a55eb5c736"
> > +
> > +  depends_on "popt" => :build
> > +  depends_on "libpng12" => :build
>
> Core formulae cannot depend on tap formulae, I'm afraid. It can be an 
> |optional| dep, but not a mandatory one. It'd also need to be fully 
> qualified, i.e. |depends_on "homebrew/versions/libpng"|, otherwise 
> it'll break.
>
Hmm. Debian has a patch to use modern libpng. Is there a way to use 
patches? If not, it looks like
we have to drop this formula.

I still use my palm to keep passwords on, as it is offline. But I'm 
probably the last person on the world
to use one, so no big loss.

Jon
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/Homebrew/homebrew/pull/38120/files#r27301898>.
>